notebooks/community/ml_ops/stage7/get_started_with_model_monitoring_xgboost.ipynb

```diff
@@ -0,0 +1,1477 @@
+{
```

Comment:
<p>Can we mention how long this approximately takes? I checked the table from the console and it's not empty but it's still printing the total_rows as 0! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1211/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='80'/>

Reply:
[screenshot oof total rows query](https://screenshot.googleplex.com/4RjYKBLxsc93SZu.png)

Reply:
<p>done</p>

Reply:
<p>The second call should already have 500 entries. We are waiting for next round which is about 1000 entries</p>